Gems/AtomLyIntegration/TechnicalArt/DccScriptingInterface/Tools/DCC/Maya/Scripts/Python/scene_exporter/readme.md

```diff
@@ -0,0 +1,51 @@
+# O3DE DCCsi, Maya Scene Exporter
+
+The "DccScriptingInterface" (aka DCCsi) is a Gem for O3DE to extend and interface with dcc tools in the python ecosystem. This document contains the details of using the 'Scene Exporter' for Maya that is provided as a utility with the DCCsi.
+
+###### Status: Prototype
+
+###### Version: 0.0.1
+
+###### OS: Windows only (for now)
+
+###### Support:
+
+- Maya 2022 w/ Python3 (will not support py2.7)
+
+- Maya 2023+ w/ Python3 <-- default version
+
+## Getting Started
+
+Before using this tool ensure that you have done the following:
+
+- this
+
+- that
+
+- and other things
+
+## Using the Scene Exporter
+
+This section describes how to utilize the Scene Export tool.
+
+### Exporting Scene Objects and FBX Models
+
+Do this ...
```

Comment:
Should this and the section below be marked as `TODO` or omitted for now?

Reply:
yes, Ben is going to write this doc in the next PR

Reply:
LYN-16100